render/vulkan/vulkan.c

```diff
@@ -33,6 +33,73 @@ static const char *find_extensions(const VkExtensionProperties *avail,
 	return NULL;
 }
 
+static const char* const *get_instance_layers(uint32_t *layer_count) {
+	// TODO request layer via env var WLR_VK_INSTANCE_LAYER
```

Comment:
The env var I mentioned doesn't need any action from wlroots. It's called `VK_INSTANCE_LAYERS` and is handled by the Vulkan loader.

Reply:
Oh that's nice, I wasn't aware of that. I'll remove the comment.